swatch-producer-aws/src/main/resources/application.properties

```diff
@@ -125,8 +125,10 @@ quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultAp
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".trust-store-password=${clowder.endpoints.swatch-contracts-service.trust-store-password}
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".trust-store-type=${clowder.endpoints.swatch-contracts-service.trust-store-type}
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".providers=com.redhat.swatch.resteasy.client.SwatchPskHeaderFilter, com.redhat.swatch.aws.resource.DefaultApiExceptionMapper, com.redhat.swatch.common.resteasy.OffsetDateTimeParamConverterProvider
-com.redhat.swatch.processors.BillableUsageProcessor/lookupAwsUsageContext/Retry/maxRetries=${AWS_USAGE_CONTEXT_LOOKUP_RETRIES}
-com.redhat.swatch.processors.BillableUsageProcessor/send/Retry/maxRetries=${AWS_SEND_RETRIES}
+# Setting up the timeout to 2 minutes instead of 30 seconds (default)
+quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".read-timeout=120000
+com.redhat.swatch.aws.service.AwsBillableUsageAggregateConsumer/lookupAwsUsageContext/Retry/maxRetries=${AWS_USAGE_CONTEXT_LOOKUP_RETRIES}
+com.redhat.swatch.aws.service.AwsBillableUsageAggregateConsumer/send/Retry/maxRetries=${AWS_SEND_RETRIES}
```

Comment:
This is why I very dislike having full qualified class names here: the class name changed a while ago and we didn't spot this. 
@kflahert I think this is a good opportunity to replace this and start using the `@RetryWithExponentialBackoff` annotation, wdyt?

Reply:
Oof yeah I didn't see this, I agree we should start using @RetryWithExponentialBackoff